src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.cs

```diff
@@ -59,6 +64,12 @@ private static async Task InitializeStandardComponentServicesAsync(HttpContext h
             authenticationStateProvider.SetAuthenticationState(Task.FromResult(authenticationState));
         }
 
+        var formData = httpContext.RequestServices.GetRequiredService<FormDataProvider>() as IHostEnvironmentFormDataProvider;
```

Comment:
```suggestion
        var formData = httpContext.RequestServices.GetRequiredService<IHostEnvironmentFormDataProvider>();
```

Reply:
We don't register `IHostEnvironmentFormDataProvider` in DI

Reply:
I think doing this would also require changing how `FormDataProvider` is registered in DI.